requirements.txt

```diff
@@ -1,6 +1,7 @@
 codecov
 pytest
 pytest-cov
+pytest-mock
```

Comment:
I'm not opposed to adding new dependencies, but can modify the first paragraph in `docs/testing.rst`? Something like:
```rst
To run the test-suite, you will need to install the `pytest <https://docs.pytest.org/en/latest/>`_ and `pytest-mock <https://pypi.python.org/pypi/pytest-mock>`_ packages.
```